randovania/layout/lib/teleporters.py

```diff
@@ -204,3 +204,25 @@ def dangerous_settings(self):
         if self.mode == TeleporterShuffleMode.ONE_WAY_ANYTHING:
             return ["One-way anywhere elevators"]
         return []
+
+
+@dataclasses.dataclass(frozen=True)
+class DreadTeleporterConfiguration(TeleporterConfiguration):
+    # Dread has only save stations as start nodes in the db. Elevators are no start nodes but
+    # are valid targets and also the only valid targets. Preset settings makes sure
+    # that nothing else is selected
```

Comment:
does the UI permit selecting transports correctly despite this?

Reply:
With the current modes I added:
![image](https://github.com/randovania/randovania/assets/117127188/91ceddde-c9bc-4e5e-bcdb-482aeb24dc72)

I removed the "anywhere" options because it looks like the following screenshot. If we want that to be included, it needs a little refactor because all save stations are removed. I think that's some prime specific code as it explicity is removing save stations? (and for dread that means only two are left).
```
...
    has_save_station = any(node.name == "Save Station" for node in area.nodes)
    return (area.has_start_node() and node in area.get_start_nodes() and not has_save_station)
...
```
Though I don't know how stable `anywhere` option is. Iirc remember there was some weird bug that you were not able to move if you never visited a save station before and took a teleporter that was leading to one.
![image](https://github.com/randovania/randovania/assets/117127188/10235d17-c565-4dc9-b6c7-1072a548987b)

Reply:
I'm down for just getting rid of anywhere for now. We should remove intro room as a valid target as well then

Reply:
See first screenshot. That's how I want to merge it. No `Valid elevator targets` section at all and no anywhere option